app/src/main/res/values/strings.xml

```diff
@@ -491,43 +491,43 @@
     <string name="ct_bs_app_info_error">App Info not found</string>
 
     <!-- Home screen -->
-    <string name="firewall_card_status_inactive">Disabled</string>
+    <string name="firewall_card_status_inactive">Off</string>
     <string name="firewall_card_status_active">%1$s apps</string>
 
     <string name="firewall_card_text_active">%1$s blocked\n%2$s bypassed\n%3$s excluded\n%4$s lockdown</string>
-    <string name="firewall_card_text_inactive">Enable firewall mode.</string>
+    <string name="firewall_card_text_inactive">Turn on firewall mode.</string>
     <string name="firewall_card_configure_btn">block apps</string>
 
-    <string name="dns_card_connected_status_failure">Disabled</string>
+    <string name="dns_card_connected_status_failure">Off</string>
 
     <string name="dns_card_latency_active">Latency: %1$sms</string>
-    <string name="dns_card_latency_inactive">Enable DNS mode.</string>
+    <string name="dns_card_latency_inactive">Turn on DNS mode.</string>
 
-    <string name="apps_card_ips_count">%1$s IP &amp; Port rules</string>
+    <string name="apps_card_ips_count">%1$s IP and port rules</string>
```

Comment:
prefer `&amp;` over `and` here, unless you think it would be troublesome for translations.

Reply:
It is, and I posit it is also wrong in English even though _the Goog recommends_.